app/src/main/java/com/sun_asterisk/music_65/screen/allsong/adapter/BannerPagerAdapter.java

```diff
@@ -0,0 +1,57 @@
+package com.sun_asterisk.music_65.screen.allsong.adapter;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v4.view.PagerAdapter;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import com.bumptech.glide.Glide;
+import com.sun_asterisk.music_65.R;
+import com.sun_asterisk.music_65.data.model.Song;
+import com.sun_asterisk.music_65.utils.CommonUtils;
+import java.util.List;
+
+public class BannerPagerAdapter extends PagerAdapter {
+    private Context mContext;
+    private List<Song> mSongList;
+
+    public BannerPagerAdapter(Context context, List<Song> songList) {
+        this.mContext = context;
+        this.mSongList = songList;
+    }
+
+    @NonNull
+    @Override
+    public Object instantiateItem(@NonNull ViewGroup container, int position) {
+        LayoutInflater inflater =
+                (LayoutInflater) mContext.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+        View bannerLayout = inflater.inflate(R.layout.item_banner_home, null);
+        ImageView imageBanner = bannerLayout.findViewById(R.id.imageBanner);
```

Comment:
tách method ra thành 1 method mới.
```suggestion
        onBindView(view)
```